ctapipe/io/containers.py

```diff
@@ -542,120 +542,44 @@ class SST1MDataContainer(DataContainer):
-        completeness of the ring
-    ring_pix_completeness:
-        pixel completeness of the ring
-    ring_num_pixel: int
-        Number of pixels composing the ring
-    ring_size:
-        ring size
-    off_ring_size:
-        size outside of the ring
-    ring_width:
-        ring width
-    ring_time_width:
-        standard deviation of the photons time arrival
-    prediction: dict
-        ndarray of the predicted charge in all pixels
-    mask:
-        ndarray of the mask used on the image for fitting
-
-    """
-
-    obs_id = DeprecatedField(
-        0, "run identification number", reason="moved to event.index"
-    )
-    event_id = DeprecatedField(
-        0, "event identification number", reason="moved to event.index"
-    )
-    tel_id = DeprecatedField(
-        0, "telescope identification number", reason="moved to event.index"
-    )
-    ring_completeness = Field(0.0, "fraction of ring present")
-    ring_pix_completeness = Field(0.0, "fraction of pixels present in the ring")
-    ring_num_pixel = Field(0, "number of pixels in the ring image")
-    ring_size = Field(0.0, "size of the ring in pe")
-    off_ring_size = Field(0.0, "image size outside of ring in pe")
-    ring_width = Field(0.0, "width of the muon ring in degrees")
-    ring_time_width = Field(0.0, "duration of the ring image sequence")
+    ring_completeness = Field(nan, "fraction of ring present")
+    ring_pix_completeness = Field(nan, "fraction of pixels present in the ring")
+    ring_num_pixel = Field(-1, "number of pixels in the ring image")
+    ring_size = Field(nan, "size of the ring in pe")
+    off_ring_size = Field(nan, "image size outside of ring in pe")
+    ring_width = Field(nan, "width of the muon ring in degrees")
+    ring_time_width = Field(nan, "duration of the ring image sequence")
     impact_parameter = Field(
-        0.0, "distance of muon impact position from centre of mirror"
+        nan, "distance of muon impact position from centre of mirror"
     )
-    impact_parameter_chi2 = Field(0.0, "impact parameter chi squared")
-    intensity_cov_matrix = Field(0.0, "covariance matrix of intensity")
-    impact_parameter_pos_x = Field(0.0, "impact parameter x position")
-    impact_parameter_pos_y = Field(0.0, "impact parameter y position")
-    COG_x = Field(0.0, "Centre of Gravity x")
-    COG_y = Field(0.0, "Centre of Gravity y")
-    prediction = Field([], "image prediction")
-    mask = Field([], "image pixel mask")
-    optical_efficiency_muon = Field(0.0, "optical efficiency muon")
-    intensity_fit_method = Field("", "intensity fit method")
-    inputfile = Field("", "input file")
+    impact_parameter_chi2 = Field(nan, "impact parameter chi squared")
+    intensity_cov_matrix = Field(nan, "covariance matrix of intensity")
+    impact_parameter_pos_x = Field(nan, "impact parameter x position")
+    impact_parameter_pos_y = Field(nan, "impact parameter y position")
+    cog_x = Field(nan, "Centre of Gravity x")
```

Comment:
changing `COG_x` -> `cog_x` may affect some of the scripts that are using these containers, but ok.

Reply:
The other stuff I changed will do the same, this is a breaking change, yes

Reply:
that will be the case in all of the container cleanups that will happen soon - so far we use only lower case elsewhere, so I would avoid doing something different here.

Reply:
By the way, are these fields the same has in the hillas parameters? surely looks the same

Reply:
`COG_x` (or `cog_x`) is likely calculated in an "uncleaned" image whose pixel_ids have been limited by their distance to the fitted ring. It is the same as `ring_size`. Since they are not calculated the same, I would still keep them separated, maybe calling them `ring_cog_x/y`?

Reply:
this will be dealt with container prefixes, a thing inside a container named `MuonRing` should not be called something like `ring_`